### PR TITLE
fix: make small changes to Concepts.md

### PR DESCRIPTION
**Pull request by shimks**
Please review the 2nd commit

## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] Related API Documentation was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

**Comment by shimks** [comment]
No, not really. I just borrowed up your setup in that other PR you had :p. I'll revert it back

File: docs/site/Concepts.md [hunk 1]
```diff
@@ -10,22 +10,40 @@ summary:
...
-- [**Sequence**](Sequence.md): A stateless grouping of [Actions](Sequence.md#actions) that control how a Server responds to requests.
-- [**Model**](Model.md): Represents the definition of a model in LoopBack, with respect to the datasource juggler. The `@loopback/repository` module provides special decorators for adding metadata to TypeScript/JavaScript classes to use them with the legacy implementation of Datasource Juggler. In addition, `@loopback/repository-json-schema` module uses the decorators' metadata to build a matching JSON Schema.
-- [**Repository**](Repositories.md): Type of Service that represents a collection of data within a DataSource.
-- [**Decorator**](Decorators.md): Enables you to annotate or modify your class declarations and members with metadata.
+- [**Application**](Application.md): The central class for setting up all of
+  your module’s components, controllers, servers and bindings. The Application
+  class extends [Context](Context.md) and provides the controls for starting and
+  stopping its associated servers.
+- [**Server**](Server.md): An implementation for inbound protocols such as REST,
```
Comment (raymondfeng) [inline docs/site/Concepts.md]: `inbound protocols` -> `inbound transports/protocols`

**Comment by shimks** [comment]
@slnode test please

**Comment by shimks** [comment]
@slnode test please

**Pull request merged**